modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view_file_entry_metadata_sets.jsp

```diff
@@ -28,7 +28,7 @@ DLViewFileEntryMetadataSetsDisplayContext dLViewFileEntryMetadataSetsDisplayCont
 />
 
 <portlet:actionURL copyCurrentRenderParameters="<%= true %>" name="/document_library/delete_data_definition" var="deleteDataDefinitionURL">
-	<portlet:param name="mvcPath" value="/view_file_entry_metadata_sets.jsp" />
+	<portlet:param name="mvcPath" value="/document_library/view_file_entry_metadata_sets.jsp" />
```

Comment:
I think you can remove this line.

Reply:
![](https://media0.giphy.com/media/xT3i1ilz3g5pP6Dq8w/giphy.gif)